<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Conjuntos</class>
 <widget class="QMainWindow" name="Conjuntos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>876</width>
    <height>554</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>554</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>876</width>
    <height>554</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: black;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>876</width>
     <height>512</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>876</width>
     <height>512</height>
    </size>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>831</width>
      <height>482</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>831</width>
      <height>482</height>
     </size>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_10">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>90</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>100</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel{
    background-color: #404040;
    color: #FFFFFF;
    border-style: outset;
    padding: 2px;
    font: bold 20px;
    border-width: 6px;
    border-radius: 10px;
    border-color: green;
}</string>
           </property>
           <property name="text">
            <string>Conjunto A:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>90</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>100</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"> QLabel{
    background-color: #404040;
    color: #FFFFFF;
    border-style: outset;
    padding: 2px;
    font: bold 20px;
    border-width: 6px;
    border-radius: 10px;
    border-color: green;
}</string>
           </property>
           <property name="text">
            <string>Conjunto B:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>90</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>100</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"> QLabel{
    background-color: #404040;
    color: #FFFFFF;
    border-style: outset;
    padding: 2px;
    font: bold 20px;
    border-width: 6px;
    border-radius: 10px;
    border-color: green;
}</string>
           </property>
           <property name="text">
            <string>Conjunto C:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLineEdit" name="txtSetA">
           <property name="minimumSize">
            <size>
             <width>680</width>
             <height>90</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>680</width>
             <height>100</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">
    background-color: #404040;
    color: #FFFFFF;
    border-style: double;
    padding: 2px;
    font: bold 20px;
    border-width: 6px;
    border-radius: 10px;
    border-color: green;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="txtSetB">
           <property name="minimumSize">
            <size>
             <width>680</width>
             <height>90</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>100</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">
    background-color: #404040;
    color: #FFFFFF;
    border-style: double;
    padding: 2px;
    font: bold 20px;
    border-width: 6px;
    border-radius: 10px;
    border-color: green;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="txtSetB_2">
           <property name="minimumSize">
            <size>
             <width>680</width>
             <height>90</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>100</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">
    background-color: #404040;
    color: #FFFFFF;
    border-style: double;
    padding: 2px;
    font: bold 20px;
    border-width: 6px;
    border-radius: 10px;
    border-color: green;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="maximumSize">
        <size>
         <width>829</width>
         <height>20</height>
        </size>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_35">
         <property name="minimumSize">
          <size>
           <width>10</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>10</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_34">
         <property name="minimumSize">
          <size>
           <width>290</width>
           <height>29</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>29</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color:white;</string>
         </property>
         <property name="text">
          <string>Selecciona el n√∫mero de conjuntos a evaluar:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton">
         <property name="styleSheet">
          <string notr="true">color: white;</string>
         </property>
         <property name="text">
          <string>Dos Conjuntos</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_2">
         <property name="styleSheet">
          <string notr="true">color:white;</string>
         </property>
         <property name="text">
          <string>Tres Conjuntos</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_40">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>29</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>29</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_41">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>29</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>29</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_42">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>29</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>29</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_43">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>29</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>29</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>29</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>29</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>29</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>29</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>29</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>29</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="minimumSize">
          <size>
           <width>110</width>
           <height>29</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>110</width>
           <height>29</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cmb1">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>71</width>
           <height>29</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>71</width>
           <height>29</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QComboBox{
  background-color: #404040;
  border: thin solid blue;
  border-radius: 4px;
  display: inline-block;
  font: inherit;
  color: #FFFFFF;
  line-height: 1.5em;
  padding: 0.5em 0.2em 0.2em 0.2em;
}

 QComboBox::down-arrow {
     image: url(/Users/benjamin/PycharmProjects/FCC_Toolkit/Designs/arrow.png);
 }</string>
         </property>
         <property name="duplicatesEnabled">
          <bool>false</bool>
         </property>
         <item>
          <property name="text">
           <string>-</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>A</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>B</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>C</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbOperador2_2">
         <property name="maximumSize">
          <size>
           <width>48</width>
           <height>29</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color: #FFFFFF;</string>
         </property>
         <property name="text">
          <string>*</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_18">
         <property name="maximumSize">
          <size>
           <width>48</width>
           <height>29</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color: #FFFFFF;</string>
         </property>
         <property name="text">
          <string>(</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cmb1_3">
         <property name="minimumSize">
          <size>
           <width>71</width>
           <height>29</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>71</width>
           <height>29</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QComboBox{
  background-color: #404040;
  border: thin solid blue;
  border-radius: 4px;
  display: inline-block;
  font: inherit;
  color: #FFFFFF;
  line-height: 1.5em;
  padding: 0.5em 0.2em 0.2em 0.2em;
}

 QComboBox::down-arrow {
     image: url(/Users/benjamin/PycharmProjects/FCC_Toolkit/Designs/arrow.png);
 }</string>
         </property>
         <item>
          <property name="text">
           <string>-</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>A</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>B</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>C</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbOperador2">
         <property name="maximumSize">
          <size>
           <width>48</width>
           <height>29</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color: #FFFFFF;</string>
         </property>
         <property name="text">
          <string>*</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cmb1_2">
         <property name="minimumSize">
          <size>
           <width>71</width>
           <height>29</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>71</width>
           <height>29</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QComboBox{
  background-color: #404040;
  border: thin solid blue;
  border-radius: 4px;
  display: inline-block;
  font: inherit;
  color: #FFFFFF;
  line-height: 1.5em;
  padding: 0.5em 0.2em 0.2em 0.2em;
}

 QComboBox::down-arrow {
     image: url(/Users/benjamin/PycharmProjects/FCC_Toolkit/Designs/arrow.png);
 }</string>
         </property>
         <item>
          <property name="text">
           <string>-</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>A</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>B</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>C</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_21">
         <property name="maximumSize">
          <size>
           <width>49</width>
           <height>29</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color: #FFFFFF;</string>
         </property>
         <property name="text">
          <string>)</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_11">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>29</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>29</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_10">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>29</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>29</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>29</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>29</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>29</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>29</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="maximumSize">
        <size>
         <width>829</width>
         <height>20</height>
        </size>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QPushButton" name="btnUnion">
           <property name="styleSheet">
            <string notr="true">QPushButton{
	font: bold 18px;
	border-style: inset;
	border-width: 2px;
    border-radius: 10px;
    border-color: green;
	color: #FFFFFF;
}

QPushButton::pressed
{
	background-color : green;
}</string>
           </property>
           <property name="text">
            <string>Uni√≥n</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnIntersection">
           <property name="maximumSize">
            <size>
             <width>203</width>
             <height>26</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
	font: bold 18px;
	border-style: inset;
	border-width: 2px;
    border-radius: 10px;
    border-color: green;
	color: #FFFFFF;
}

QPushButton::pressed
{
	background-color : green;
}</string>
           </property>
           <property name="text">
            <string>Intersecci√≥n</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnDifference">
           <property name="maximumSize">
            <size>
             <width>202</width>
             <height>26</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
	font: bold 18px;
	border-style: inset;
	border-width: 2px;
    border-radius: 10px;
    border-color: green;
	color: #FFFFFF;
}

QPushButton::pressed
{
	background-color : green;
}</string>
           </property>
           <property name="text">
            <string>Diferencia</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnSymmetry">
           <property name="maximumSize">
            <size>
             <width>202</width>
             <height>26</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
	font: bold 18px;
	border-style: inset;
	border-width: 2px;
    border-radius: 10px;
    border-color: green;
	color: #FFFFFF;
}

QPushButton::pressed
{
	background-color : green;
}</string>
           </property>
           <property name="text">
            <string>Diferencia Sim√©trica</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>876</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
